src/service/crypto-impl.js

```diff
@@ -213,6 +213,41 @@ export class Crypto {
       data
     ));
   }
+
+  /**
+   * Returns a cryptographically random string with 128 bits of entropy
+   * to be used as a subdomain for SafeFrame.
```

Comment:
Let's not reference specific usage case: subdomain for Safeframe.

Reply:
removed